### PR TITLE
LKE-1065 Cloud Manager filters regions to only those regions that have LKE

### PR DESCRIPTION
**Pull request by WilkinsKa1**
## Description

We were previously displaying all DCs as available in k8 creation, however if you tried to create a cluster in a DC without LKE capability, it would throw an error. Now we filter based on capability so users won't be able to attempt an action that will only result in error. 

## Type of Change
- Non breaking change ('change')

## Note to Reviewers

Please test (for now) using staging.

### CODE REVIEW

File: packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx [hunk 1]
```diff
@@ -221,8 +221,15 @@ export class CreateCluster extends React.Component<CombinedProps, State> {
...
-    const _region = regionsData
-      ? regionsData.find(thisRegion => thisRegion.id === selectedRegion)
+    // Only displaying regions that have LKE capability
+    const filteredRegions = regionsData
+      ? regionsData.filter(thisRegion =>
+          thisRegion.capabilities.includes('Kubernetes')
+        )
+      : undefined;
+
+    const _region = filteredRegions
```
Comment (WilkinsKa1) [inline packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx]: @Jskobos Do you think i should remove the check for `filteredRegions` here? I'm checking the existence of `regionsData` above so seems redundant, but perhaps its safer jic.

Comment (Jskobos) [reply packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx]: (then remove the `|| []` below)

File: packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx [hunk 2]
```diff
@@ -248,14 +255,14 @@ export class CreateCluster extends React.Component<CombinedProps, State> {
             <SelectRegionPanel
               error={errorMap.region}
               copy={'Determine the best location for your cluster.'}
-              regions={regionsData || []}
+              regions={filteredRegions || []}
```
Comment (Jskobos) [inline packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx]: this is redundant now

Comment (Jskobos) [reply packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx]: (the `|| []`)

**Pull request merged**